frontend/src/components/samples/details/SampleDetailsProcessMeasurements.js

```diff
@@ -48,7 +48,7 @@ const SampleDetailsProcessMeasurements = ({processMeasurements, usersByID, proto
         align: 'right',
         render: (volumeUsed, _) =>
             <div>
-              {volumeUsed ? `${parseFloat(volumeUsed).toFixed(3)}` : undefined}
+              {parseFloat(volumeUsed).toFixed(3)}
```

Comment:
should it also test against null?

Reply:
Not sure what can set a variable to null ... but yeah possibly ... I don't think we'll knowingly set it to null but it is another real false.

Reply:
Actually, it will be null if no value has been set, 0 if the volume is zero, or undefined if the sample is still being fetched and it completely missing the volume property. Maybe we should write a `renderFloat()` function to take care of all that complexity.

Reply:
Nah I believe we should add more bloat by installing another package to take care of that 😉

Reply:
Also what should `renderFloat()` show if they're `undefined` or `null`.